### ISSUE TITLE
Error! Unhandled error (execa).

### ISSUE THREAD

**Issue by TrukiSt** [jely2002/youtube-dl-gui#493]
Hello. I'm pretty tired of Open Video Downloader because everytime there's a diferent  error in this program. Now it's **Error! Unhandled error (execa)**.

Tried to search how to solve it but couldn't find anything that really worked. The program and yt-dlp are in the lastest version.

![image](https://github.com/jely2002/youtube-dl-gui/assets/129324146/f4d2995d-53d8-4634-be53-d013e8ecccc8)
![image](https://github.com/jely2002/youtube-dl-gui/assets/129324146/3949f3e1-396d-422b-94a7-144480359e54)

Help please!

**Comment by 42willow** [comment]
Try using https://github.com/StefanLobbenmeier/youtube-dl-gui for now. It works flawlessly.

**Comment by Yuban9954** [comment]
ME TOOOOO！

**Comment by 1000goalgaming** [comment]
I agree. I usually end up using the shady websites for this stuff because it seems like this app never works properly. Have tried nearly everything I've come across from looking around various issues on this repo that I've come across to fix the multitude of problems I've had with this app in the past.

**Comment by himyjan** [comment]
https://github.com/jely2002/youtube-dl-gui/issues/354#issuecomment-1127269708

**Comment by rahadiannorman** [comment]
> [#354 (comment)](https://github.com/jely2002/youtube-dl-gui/issues/354#issuecomment-1127269708)
> 
> or
> 
> https://github.com/StefanLobbenmeier/youtube-dl-gui/releases

This one finally works for me, thanks!

**Comment by jely2002** [comment]
Closing this issue as it has been fixed in v3 of this app.
